web/src/utils/formatting.ts

```diff
@@ -17,7 +17,7 @@ export const formatPower = function (
   if (d == undefined || Number.isNaN(d)) {
     return d;
   }
-  const power = `${d3.format(`.${numberDigits}s`)(d * 1e6)}Wh`;
+  const power = `${d3.format(`.${numberDigits}s`)(d)}Wh`;
   return addSpaceBetweenNumberAndUnit(power);
```

Comment:
Just checked, this is definitely needed - otherwise we show kWh instead of GWh.

Actual:
![Screenshot 2023-09-07 at 13 08 01](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/1bf0714e-59f4-4c70-ad54-5a97197ec012)

Expected:
![Screenshot 2023-09-07 at 13 10 02](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/f94c8226-1432-4bea-b407-bfa860e8fc43)